garrysmod/gamemodes/terrortown/gamemode/weaponry.lua

```diff
@@ -425,11 +425,17 @@ local function OrderEquipment(ply, cmd, args)
 end
 concommand.Add("ttt_order_equipment", OrderEquipment)
 
+function GM:TTTToggleDisguiser(ply, state)
+   -- Can be used to prevent players from using this button.
+   -- return false to prevent it.
+end
+
 local function SetDisguise(ply, cmd, args)
    if not IsValid(ply) or not ply:IsActiveTraitor() then return end
 
    if ply:HasEquipmentItem(EQUIP_DISGUISE) then
       local state = #args == 1 and tobool(args[1])
+      if not hook.Run("TTTToggleDisguiser", ply, state) == false then return end

```

Comment:
Can it be that I have the disguised? I'll test this again later.
(Because I'm checking if the hook returns anything other then false. So if it returns nil it wouldn't work, would it?)